dali/pipeline/data/types.h

```diff
@@ -33,17 +33,7 @@
 #include "dali/core/error_handling.h"
 #include "dali/core/tensor_layout.h"
 
-// Workaround missing "is_trivially_copyable" in libstdc++ for g++ < 5.0.
-// We have to first include some standard library headers, so to have __GLIBCXX__ symbol,
-// and we have to exclude the specific version used in manylinux for our CI, because
-// clang always defines __GNUC__ to 4. __GLIBCXX__ is not a linear ordering based on
-// version of library but the date of the release so we can have 4.9.4 > 5.1.
-#if __GLIBCXX__ == 20150212 || (__cplusplus && __GNUC__ < 5 && !__clang__)
-#include <boost/type_traits/has_trivial_copy.hpp>
-#define IS_TRIVIALLY_COPYABLE(T) ::boost::has_trivial_copy<T>::value
-#else
 #define IS_TRIVIALLY_COPYABLE(T) std::is_trivially_copyable<T>::value
```

Comment:
TBH, you could remove this macro as well.

Reply:
done